sharepoint-server-farm-ha/azuredeploy.json

```diff
@@ -0,0 +1,1985 @@
+{
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"newStorageAccountName": {
```

Comment:
Good feedback, I don't think we are going to hit any issues as there are only 11 disks in use but it wont do any harm - other than use quota - to separate storage accounts for AD, SQL and SP. I plan to add the capability to specify the number of SP VMs once DSC is fixed so this will help there.